doc/OnlineDocs/tutorial/index.rst

```diff
@@ -29,6 +29,72 @@ Concrete vs Abstract Models
 
 Modeling Components
 -------------------
+The primary modeling components defined by Pyomo are
+:class:`Set<pyomo.core.base.sets.Set>`, :class:`Var<pyomo.core.base.var.Var>`,
```

Comment:
Did you intentionally omit Param?  If so, then I think it makes sense to also omit Set.  Both Set and Param provide data abstractions that are not strictly necessary to build Pyomo models.